core/src/main/java/io/questdb/griffin/engine/functions/regex/MatchVarcharFunctionFactory.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin.engine.functions.regex;
+
+/**
+ * This is tactical implementation of regex match over varchar column.
+ * It expoits the ability of a varchar column to return a CharSequence view of the sequence.
+ */
+public class MatchVarcharFunctionFactory extends MatchStrFunctionFactory {
```

Comment:
We should have a similar stub implementation for `RegexpReplaceStrFunctionFactory`

Reply:
Done in [eb12311](https://github.com/questdb/questdb/pull/4308/commits/eb12311eb4702bdfd6783c65317580ca5e4299dc)